### PR TITLE
Clarify that triples may be ignored/discarded

### LINKED ISSUE #350: Clarify that normative triples that the Verifier does not understand MAY be ignored
There will be extensions in later specs, and there are some heavy base spec triples that may not always be needed. We should state what the expected behavior is.

### PR DESCRIPTION
**Pull request by deeglaze**
Fixes #350

### CODE REVIEW

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -1506,6 +1506,8 @@ The CoRIM profile MUST provide a description of the expected Verifier behavior f
 
 Verifier implementations MUST exhibit the same externally visible behavior as described in this specification.
 They are not required to use the same internal representation or evaluation order described by this specification.
+They are not required to implement support for all triples, profile permitting.
```
Comment (yogeshbdeshpande) [inline draft-ietf-rats-corim.md:1509]:
```suggestion
A Verifier may not implement all the Triples specified in this specification. It depends on its Appraisal Policy for Evidence as mandated by its profile.
```

File: draft-ietf-rats-corim.md [hunk 2]
```diff
@@ -1506,6 +1506,8 @@ The CoRIM profile MUST provide a description of the expected Verifier behavior f
...
+A Verifier MAY discard a triple based on policy.
```
Comment (yogeshbdeshpande) [inline draft-ietf-rats-corim.md:1510]:
```suggestion

```

**Comment by deeglaze** [comment]
Discarding triples is left to later phases and not normative for up to phase 4.
Ignoring new triples is a matter of what is NTI, which @henkbirkholz is going to write what is NTI.

File: draft-ietf-rats-corim.md (hunk 1 above)
Comment (yogeshbdeshpande) [reply draft-ietf-rats-corim.md:1509]: Pending M T I work!

**Pull request closed without merging**